docs/src/test-assertions-js.md

```diff
@@ -134,7 +134,7 @@ expect(test.info().errors).toHaveLength(0);
 
 Note that soft assertions only work with Playwright test runner.
 
-## Custom Expect Message
+## Custom expect message
```

Comment:
Why are these changing? The style was consistent in this document and we use both across the docs.